pygmt/src/logo.py

```diff
@@ -39,13 +39,14 @@ def logo(self, **kwargs):
     {J}
     {R}
     position : str
-        ``'[g|j|J|n|x]refpoint+wwidth[+jjustify][+odx[/dy]]'``.
+        [**g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**]\ *refpoint*
+        **+w**\ *width*\ [**+j**\ *justify*]\ [**+o**\ *dx*\ [/*dy*]].
         Sets reference point on the map for the image.
     box : bool or str
         Without further options, draws a rectangular border around the
         GMT logo.
     style : str
-        ``l|n|u``.
+        [**l**\|\ **n**\|\ **u**].
```

Comment:
It's not clear to me why there is a return after this argument in the docs, after (str) -, when there isn't one for the other modules. It's in the [current docs](https://www.pygmt.org/dev/api/generated/pygmt.Figure.logo.html) too. If anyone has a recommendation for why this happens, please let me know.

Reply:
It's also unclear to me. We use the [napolen](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) extension for the docstring, but I don't know how napolen works.

Reply:
Napolean sphinx was added in #383, and we unpinned the sphinx version in #629 in Dec 2020. Not sure if that last PR did anything, I didn't notice any change until we were working on the docs recently (i.e. this week).

Reply:
Thanks, I figure out that it happens anytime there is a bullet list in the argument. But, I don't know if there's a way to stop that from happening. Will leave it for now.